root/static/manifest.mjs

```diff
@@ -17,16 +15,29 @@
  * there. The actual manifest file (./build/rev-manifest) doesn't exist
  * until after the client scripts are bundled.
```

Comment:
It's still true, we're just reading it differently now (`readFileSync` vs. `require`).  Because `require` isn't available in an ES module (unless you create it, but we don't need to).